### ISSUE TITLE
URL deletion

### ISSUE THREAD

**Issue by bryanpedini** [adyanth/shortpaste#31]
Hello,

It would be really helpful to be able to delete old / unused / rotten URLs, to be able to reuse some short-links, to update the URL.

KR,
Bryan.

**Comment by adyanth** [comment]
Deleting a URL in a URL shortener is generally a bad idea. One example is where URL malware scanners cache the URL to be healthy or not, and a bait and switch can be performed later if URL deletion is allowed. If you **have** to do it, you can always delete the entry in the SQlite DB, but as I said, that is usually not recommended.

**Comment by bryanpedini** [comment]
True, if you plan to leave the URL shortener public and open without any account, audit, or consequences...

False if you plan to use it yourself and/or a couple of friend, or if there are accounts with audit enabled and in case of unauthorized / malware use, you can ban an entire domain from registering an account (you ban the free mail services and all the free .tk & .ml domains, and force users to have their own domain, you see how registering fake accounts start becoming expensive)

**Comment by adyanth** [comment]
Sorry, I do not think I understand. This software does not have the concept of user accounts or sign ins. Just authentication/authorization for admin login. The protection is more for the receiver of the shortened link where their email provider/antivirus would have scanned the link and deemed it safe.

But if you are using it internally, I guess that does not matter, and reuse of links do become useful. Can be a toggle setting to enable/disable that.

**Comment by 9021007** [comment]
Any update on this? Thank you so much
